mixxx should never download external libdjinterop if the local version doesn't
meet the expected version. The libdjinterop package should be adapted instead
---
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2841,11 +2841,8 @@ if(ENGINEPRIME)
     find_package(DjInterop ${LIBDJINTEROP_VERSION} EXACT MODULE)
   endif()
 
-  if(DjInterop_FOUND)
-    # An existing installation of djinterop is available.
-    message(STATUS "STATIC link existing system installation of libdjinterop")
-    target_link_libraries(mixxx-lib PUBLIC DjInterop::DjInterop)
-  else()
+  target_link_libraries(mixxx-lib PUBLIC DjInterop::DjInterop)
+  if(FALSE)
     # On MacOS, Mixxx does not use system SQLite, so we will use libdjinterop's
     # embedded SQLite in such a case.
     if(APPLE AND NOT SQLite3_IS_STATIC)
-- 
2.49.0

